wallet/src/Pos/Wallet/Web/Mode.hs

```diff
@@ -156,6 +157,16 @@ type MonadWalletWebMode m =
     , m ~ WalletWebMode
     )
 
+-- AJ: Need to use unblocking getCurrentSlot? Also using 0 as default??
+getIsBootstrapEra ::
+    ( MonadSlots ctx m
+    , MonadGState m
+    )
+    => m IsBootstrapEraAddr
+getIsBootstrapEra = do
+    e <- maybe 0 siEpoch <$> getCurrentSlot
```

Comment:
I'm not sure if it's okay to use 0 slot as default here. This would mean that if we couldn't get current slot in reward era for some reason, we'll create addresses with bootstrap distribution automatically, which is incorrect. I think that there's a better way to detect current era without blocking. cc @gromakovsky